Document-Processing/PowerPoint/PowerPoint-Library/NET/Feature-Matrix.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Feature Matrix in .NET PowerPoint Library | Syncfusion
+description: Explore the supported features of Syncfusion® Essential PowerPoint for working with PowerPoint 2007–2016 presentations programmatically.
+platform: document-processing
+control: Presentation
+documentation: UG
+---
+
+# Feature Matrix in .NET PowerPoint Library
+
+Syncfusion® Essential PowerPoint is a [.NET Core PowerPoint library](syncfusion.com/document-processing/powerpoint-framework/net-core) that provides comprehensive support for creating, editing, and managing PowerPoint presentations programmatically. It allows seamless handling of slides, shapes, charts, SmartArt, tables, and more.
+
+The following table outlines the supported features for PowerPoint 2007–2016 presentations.
+
+## Document Content Features
+
+<table>
+  <tr>
+    <td>
+      {{'**Document Content Features**'| markdownify }}
+    </td>
+    <td>
+      {{'**PowerPoint 2007–2016**'| markdownify }}
+    </td>
+  </tr>
+  <tr><td>Create PowerPoint presentations from scratch or edit an existing presentation.</td><td> ![Yes](Feature_Matrix/yes.png) </td></tr>
+  <tr><td>Open existing PowerPoint presentations from the file system or stream.</td><td> ![Yes](Feature_Matrix/yes.png) </td></tr>
+  <tr><td>Save PowerPoint presentation to a local file, stream, or stream it to a client browser.</td><td> ![Yes](Feature_Matrix/yes.png) </td></tr>
+  <tr><td>Create, access, and modify paragraphs and text.</td><td> ![Yes](Feature_Matrix/yes.png) </td></tr>
+  <tr><td>Create, access, and modify bullets and numbering.</td><td> ![Yes](Feature_Matrix/yes.png) </td></tr>
+  <tr><td>Create, access, and modify images.</td><td> ![Yes](Feature_Matrix/yes.png) </td></tr>
+  <tr><td>Create, access, and modify tables, rows, and cells.</td><td> ![Yes](Feature_Matrix/yes.png) </td></tr>
+  <tr><td>Create, access, and modify shapes.</td><td> ![Yes](Feature_Matrix/yes.png) </td></tr>
+  <tr><td>Create, access, and modify charts.</td><td> ![Yes](Feature_Matrix/yes.png) </td></tr>
+  <tr><td>Access and modify placeholders.</td><td> ![Yes](Feature_Matrix/yes.png) </td></tr>
+  <tr><td>Create, access, and modify SmartArt diagrams.</td><td> ![Yes](Feature_Matrix/yes.png) </td></tr>
+  <tr><td>Create, access, and modify sections.</td><td> ![Yes](Feature_Matrix/yes.png) </td></tr>
```

Comment:
Include below content,

Create, access, and modify animation and slide transition.